dbw/node_fw/mod/throttle/throttle.c

```diff
@@ -69,42 +52,21 @@ static void throttle_init()
     control_relay(0);
 
```

Comment:
We don't need this define here anymore.

Reply:
Which define do you mean? The line above your comment is `control_relay(0);`.

Reply:
`CMD_TIMEOUT_MS`